### ISSUE TITLE
HTTPs?

### ISSUE THREAD

**Issue by eFlowMedia** [codestudiollc/laravel-totem#374]
So Ive set this up quite a few times, and every time I make sure that the site APP_URL="https://somedomain.com" prior to publishing and copying assets. (Note the https) But when I go to manually execute a task I get this error...

app.js:7 Mixed Content: The page at 'https://....com/totem/tasks' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http:/....com/totem/tasks/1/execute'. This request has been blocked; the content must be served over HTTPS.

How do I force totem to use https?  Do I need to do something before I install it to tell it to use https?

**Comment by eFlowMedia** [comment]
@qschmick Can you help with this one?

**Comment by qschmick** [comment]
@eFlowMedia Is you value:
```
APP_URL="somedomain.com"
```

or

```
APP_URL="https://somedomain.com"
```

It should be the latter. This is a Laravel thing

**Comment by eFlowMedia** [comment]
Its - APP_URL="https://somedomain.com"

The crons are working, its just when I try to run them manually the link goes to http instead of https - ex "http://domain//totem/tasks/2/execute"
![Screenshot 2024-03-22 at 6 20 00 AM](https://github.com/codestudiohq/laravel-totem/assets/16480031/d6c84541-9af8-4b45-9d6d-8d129efff183)

**Comment by johnnyang0815** [comment]
hi, is there any solution for this? I faced this issue as well

**Comment by eFlowMedia** [comment]
Yes, I found the issue was because we were using Cloudflare.  The solution is here...
https://laravel.com/docs/10.x/requests#configuring-trusted-proxies

I had to use "*" in order to get it to work, but solved my problem.
